internal/driver/commands.go

```diff
@@ -246,6 +246,22 @@ func helpText(s ...string) string {
 	return strings.Join(s, "\n") + "\n"
 }
 
+func variablesPostFix(name string, vr *variable) string {
```

Comment:
The name of this function is unclear. I don't understand what it's doing from the name.

Reply:
Done.